packages/pie-toolbox/src/code/math-rendering-accessible/render-math.js

```diff
@@ -331,7 +331,7 @@ export const initializeMathJax = (callback) => {
         },
       },
       loader: {
-        load: ["input/mml"],
+        load: ["input/mml", '[mml]/mml3'],
```

Comment:
Would it make sense to add error handling here? For instance:

`MathJax = {
  loader: {
    load: ["input/mml", "[mml]/mml3"],
    failed: function (error) {
      console.error("MathJax load error:", error);
    },
  },
};
`

Reply:
Thanks. I used https://docs.mathjax.org/en/v3.2-latest/options/startup/loader.html#the-configuration-block.